indigo.ignored

```diff
@@ -1,3 +1,2 @@
-chomp_motion_planner
```

Comment:
I wouldn't change Indigo.

Reply:
Got it. I'll fix once I get back to computer.